src/main/java/org/springframework/samples/petclinic/owner/PetController.java

```diff
@@ -105,6 +105,11 @@ public String initCreationForm(Owner owner, ModelMap model) {
 	@PostMapping("/pets/new")
 	public String processCreationForm(Owner owner, @Valid Pet pet, BindingResult result,
 			RedirectAttributes redirectAttributes) {
+		// Validate empty name
+		if (!StringUtils.hasText(pet.getName())) {
+			result.rejectValue("name", "required", "Pet name must not be empty");
```

Comment:
It looks like "required" already has a message associated with it, and it doesn't start with a capital letter - the third argument here is probably redundant?